packages/engine/src/physics/systems/PhysicsSystem.ts

```diff
@@ -79,24 +78,6 @@ export default async function PhysicsSystem(world: World) {
   await world.physics.createScene()
 
   return () => {
```

Comment:
what might be wise here, is to move all of the enter and exit queries to the start of the system loop, and move all the simulation logic to the end of the system loop, and before the simulation logic with an `if (Engine.isEditor) return`